src/tabs/Pixnflix/index.tsx

```diff
@@ -338,14 +381,34 @@ class PixNFlix extends React.Component<Props, State> {
             }}
             width={DEFAULT_WIDTH}
             height={DEFAULT_HEIGHT}
+            style={{ display: !isAccepting ? 'initial' : 'none' }}
           />
           <br />
-          <input
-            type='file'
-            onChange={this.handleFileUpload}
-            style={{ display: useLocalVideo ? 'initial' : 'none' }}
-          />
-          <p style={{ fontFamily: 'courier' }}>
+          <div style={{ display: isAccepting ? 'inherit' : 'none' }}>
+            <div style={{ fontSize: 40 }}>Drag file here</div>
+            <br />
+            <input type='file' onChange={this.handleFileUpload} />
+          </div>
+          <br />
+          <div
+            style={{ display: hasAudio && !isAccepting ? 'inherit' : 'none' }}
+          >
+            Volume:
+            <input
+              type='range'
+              onChange={this.handleVolumeChange}
+              min={0}
+              max={1}
+              value={volume}
+              step={0.01}
+            />
+          </div>
+          <p
+            style={{
+              display: displayOptions ? 'inherit' : 'none',
+              fontFamily: 'courier',

```

Comment:
Unsure if keeping the monospaced font is the best choice for the hint text, as opposed to a default sans serif font with some styling if needed